Content.Client/UserInterface/RichText/MonoTag.cs

```diff
@@ -0,0 +1,34 @@
+using System.Linq;
+using Robust.Client.ResourceManagement;
+using Robust.Client.UserInterface.RichText;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Client.UserInterface.RichText;
+
+/// <summary>
+/// Sets the font to a monospaced variant
+/// </summary>
+public sealed class MonoTag : IMarkupTag
+{
+    public const string MonoFont = "Monospace";
```

Comment:
This should use `[ValidatePrototypeId<FontPrototype>]`

Reply:
Done!